src/modules/network/components/networkSwitcherDropdown/NetworkSwitcherDropdown.js

```diff
@@ -121,15 +82,22 @@ function NetworkSwitcherDropdown({ noLabel, onNetworkSwitchSuccess }) {
             })}
         </MenuSelect>
       </div>
-      {blockchainAppsMeta.isError && !blockchainAppsMeta.isFetching && (
-        <div>
+      {networkStatus.isError && networkStatus.isFetched && (
+        <div className={styles.connectionFailedBlock}>
           <span>
             {t('Failed to connect to network!  ')}
-            <span onClick={blockchainAppsMeta.refetch}>{t('Try again')}</span>
+            <span onClick={networkStatus.refetch}>{t('Try again')}</span>
```

Comment:
maybe the `Try again` can be aligned to right side, currently its sandwiched

Reply:
Design is next after another. But should probably have an underline if next to eachother:
1. 
<img width="390" alt="image" src="https://user-images.githubusercontent.com/8784876/230106519-c86617ac-ba74-4f38-97a3-d6f1d1483d54.png">

2. 
<img width="396" alt="image" src="https://user-images.githubusercontent.com/8784876/230106712-e853347a-44c8-4e68-9380-9a08a3c5e51e.png">


One or two?

Reply:
1